region/norfair/crocomire/Post Crocomire Jump Room.json

```diff
@@ -1054,6 +1065,16 @@
         "Then you need to manipulate it to go high enough to be used as a stepping stone once frozen."
       ]
     },
+    {
+      "link": [5, 1],
+      "name": "Use Flash Suit",
+      "requires": [
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 22, "excessFrames": 13}}
+      ],
+      "flashSuitChecked": true,
+      "note": "Use a flash suit to spark horizontally through the speed blocks. Samus may stop above the acid, but it should always be possible to land on one of the platforms."
```

Comment:
note: Jump back from the leftmost floating platform and diagonally shinespark at the height of the jump towards the door to minimize shinespark damage.